File: 2480-find-subarrays-with-equal-sum/find-subarrays-with-equal-sum.py
class Solution:
    def findSubarrays(self, nums: List[int]) -> bool:
        st = set()
        for i in range(len(nums)-1):
            sum = nums[i] + nums[i+1]
            if sum in st:
                return True
            else:
                st.add(sum)

        return False

File: 2480-find-subarrays-with-equal-sum/find-subarrays-with-equal-sum.cpp
class Solution {
public:
    bool findSubarrays(vector<int>& nums) {
        unordered_set<int> st;
        int n = nums.size();
        for(int i = 0; i<n-1; i++){
            int sum = nums[i] + nums[i+1];
            if(st.find(sum) == st.end()){
                st.insert(sum);
            }else{
                return true;
            }
        }
        return false;
    }
};